src/allotropy/allotrope/models/adm/solution_analyzer/rec/_2024/_03/solution_analyzer.py

```diff
@@ -1,6 +1,6 @@
 # generated by datamodel-codegen:
 #   filename:  solution-analyzer.schema.json
-#   timestamp: 2024-07-19T01:24:18+00:00
+#   timestamp: 2024-08-13T06:10:53+00:00
```

Comment:
NOTE to reviewer: this is just some moving around of classes for some reason, plus using `TIntegerValue`